### ISSUE TITLE
Postman web version in the "About" tab is showing an older Firefox version

### ISSUE THREAD

**Issue by donislawdev** [a85/POSTMan-Chrome-Extension#10993]
### Is there an existing issue for this?

- [X] I have searched the tracker for existing similar issues and I know that duplicates will be closed

### Describe the Issue

Postman in the "About" tab is showing a lower version of Firefox than the actual version used (Firefox version 101 in Postman "About" tab is shown as 91 version).

### Steps To Reproduce

1. Click the settings button
2. Select "Settings" from the dropdown list
3. In the settings tab, go to the "about" tab.
4. Compare Firefox version in "about" tab with actual Firefox version used.

### Screenshots or Videos

![FirefoxVersionInPostman](https://user-images.githubusercontent.com/26207295/173302441-22fa460c-fe6a-4597-8e73-500517cb85b4.png)

### Operating System

Windows

### Postman Version

9.21.2-220607-0647

### Postman Platform

Postman Web

**Comment by timbochamp** [comment]
Hi @donislawdev, thanks for reporting! I cannot reproduce this on Postman for Web version 9.22.4 and Firefox 101.0.1. Can you please confirm that you still see this problem on these versions?
<img width="1291" alt="Screen Shot 2022-06-23 at 1 23 56 PM" src="https://user-images.githubusercontent.com/4521134/175347996-9b1a04b0-915b-411f-8ef4-adfaadb67f30.png">

**Comment by donislawdev** [comment]
Hi @timbochamp 
For me, this issue still occurs, you are using 32-bit Firefox, mine is 64-bit. Unfortunately, on my machine I can't install Firefox 32-bit, so I can't check if it would work on 32-bit version. 
![StillTheSame](https://user-images.githubusercontent.com/26207295/175506459-6fb02d86-9962-4345-8da8-d7028f0c0d46.png)

**Comment by timbochamp** [comment]
Ah, good catch! I tried on 64-bit Firefox and I see Firefox 101 in my Postman settings > About. Wondering if this has to do with the 9.23 web version that was released if you could re-verify it?
![Screen Shot 2022-06-24 at 11 39 16 AM](https://user-images.githubusercontent.com/4521134/175558924-ce5eead3-68fc-47ca-bf6d-b57ca3cc2112.png)

**Comment by donislawdev** [comment]
@timbochamp 
OK, I tested it and on my Firefox the issue still occurs, not sure why, maybe it's because I installed Firefox like 10-15 years ago? It's a very old version, I was moving it from various Windows versions. Maybe that's why. 
![StillTheSame2](https://user-images.githubusercontent.com/26207295/175579722-19dd44d2-55a5-44ac-afbc-f9929aabcf13.png)

But I tested a portable Firefox (same machine), and Postman shows actual version, I tested it on different machine + same machine, but with Linux, and all shows actual Firefox version. The issue occurs only on my Windows machine with my old Firefox.
![Fixed](https://user-images.githubusercontent.com/26207295/175581531-a3722594-55b6-4c86-a44d-f56206c082fc.png)

I think the issue is not reproducible because it occurs only on one machine with very old Firefox, I checked it out, and I can't reproduce it on different machine, different OS or same machine/OS but with portable Firefox. I think reinstalling Firefox would help, but I will skip this one.  Furthermore, I think I can close this one, thanks a lot for help :-)
